src/getImports/babelParser.js

```diff
@@ -26,13 +30,39 @@ export function getPackages(fileName, source, language) {
   const packages = [];
   const visitor = {
     ImportDeclaration({ node }) {
-      packages.push({
-        fileName,
-        loc: node.source.loc,
-        name: node.source.value,
-        line: node.loc.end.line,
-        string: compileImportString(node),
-      });
+      const configuration = workspace.getConfiguration('vulnCost');
+      let pathIgnored = false;
+
+      for (let i = 0; i < configuration.ignorePaths.length; i++) {
+        const path = configuration.ignorePaths[i];
+        pathIgnored = new RegExp(path).test(node.source.value);
+
+        if (pathIgnored) {
+          continue;
+        }
+      }
+
+      if (pathIgnored) {
+        logger.log(`Import ${node.source.value} matched ignored path: ${path}`);
+        return;
+      }
+
+      const target = path.dirname(fileName) + path.sep + node.source.value;
+      const fileExists =
+        fs.existsSync(target) ||
+        fs.existsSync(target + '.js') ||
```

Comment:
Please make the extensions configurable - think of `.jsx`, `.ts`, etc. Thanks!

Reply:
Excellent point @erwinpleo, we already expose `typescriptExtensions` and `javascriptExtensions` as configuration options so we should reuse these

Reply:
Proposed implementation here:
https://github.com/snyk/vulncost/pull/16/commits/927e53ccbc5a4213ac3057d718a9bcda80b245cb